freqtrade/pairlist/pairlistmanager.py

```diff
@@ -94,3 +96,17 @@ def refresh_pairlist(self) -> None:
         pairlist = IPairList.verify_blacklist(pairlist, self.blacklist, True)
 
         self._whitelist = pairlist
+
+    def _prepare_whitelist(self, pairlist: List[str], tickers) -> List[str]:
+        """
+        Prepare pairlist for Pairlist Filters that use tickers data - remove
+        pairs that do not have ticker available
+        """
+        if self._tickers_needed:
+            # Copy list since we're modifying this list
+            for p in deepcopy(pairlist):
+                ticker = tickers.get(p)
+                if not ticker:
```

Comment:
this can be simplified as follows:

```suggestion
                if p not in tickers:
```

The only case where your approach would catch a problem would be the following:
```
{ 
'ETH/BTC': {},
'OTHER/PAIR': {'real': 123}
}
```

However, if that were the case, then ccxt would have messed up, and i'm yet to see such a return value..